web_src/js/index.js

```diff
@@ -2243,7 +2243,7 @@ function u2fError(errorType) {
     2: $('#u2f-error-2'),
     3: $('#u2f-error-3'),
     4: $('#u2f-error-4'),
-    5: $('.u2f-error-5')
+    5: $('.u2f_error_5')
```

Comment:
is this inconsistency really right?

Reply:
yes look at line 29 in u2f_error.tmpl

Reply:
wouldn't it be better to change L29 to `u2f-error-5` instead to retain consistency?